regparse.c

```diff
@@ -763,9 +763,9 @@ names_clear(regex_t* reg)
 	e->back_refs = (int* )NULL;
       }
     }
-    if (IS_NOT_NULL(t->e)) {
-      xfree(t->e);
-      t->e = NULL;
+
+    xfree(t->e);
+    t->e = NULL;
     }
```

Comment:
```suggestion
```

We don't build this code, but you need to remove this brace to remove the if wrapping.